src/ReactiveDomain/ReactiveDomain.Tests/Logging/with_event_store.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.IO;
+using System.Net;
+using System.Threading;
+using EventStore.ClientAPI;
+using ReactiveDomain.Domain;
+using ReactiveDomain.EventStore;
+using ReactiveDomain.Tests.Specifications;
+
+namespace ReactiveDomain.Tests.Logging
+{
+    // ReSharper disable once InconsistentNaming
+    public abstract class with_event_store :
+        CommandBusSpecification,
+        IDisposable
+    {
+        protected EventStoreLoader EventStore;
+        protected IEventStoreConnection TcpEndPointConnection;
+        protected IRepository Repo;
+
+        private static readonly IPEndPoint IntegrationTestTcpEndPoint;
+
+        static with_event_store()
+        {
+            BootStrap.Load();
+            IntegrationTestTcpEndPoint = new IPEndPoint(IPAddress.Loopback, 1113);
+        }
+
+        protected override void Given()
+        {
+
```

Comment:
This should start with `base.Given();`

Reply:
CommandBusSpecification only  has an abstract Given. The above is only true if you inherit from MockRepositorySpecification - as most of our tests do.

Reply:
Ah, ok